test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,99 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> 
+-- 3. HMI send responce on SDL with invalid value of valid param and other valid params with valid values.
+-- 4. SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
```

Comment:
@Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3."SDL respond with <success = false, resultCode = "GENERIC_ERROR">  to mobile app" Should be in expected result.